detect-monero-demo/README.md

```diff
@@ -0,0 +1,67 @@
+# Detect Monero Miners with bpftrace
+This demo accompanies the [Detect Monero Miners with bpftrace blogpost](https://blog.px.dev/detecting-monero-miners).
+## Optional: Deploy k0s
```

Comment:
I would probably have this rephrased as a Prereqs section

1. (Optional) Deploy K0s 
2. Deploy Pixie (link to install guide)